examples/jetson-nano_blink.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
```

Comment:
please add to all examples (see other examples):

// +build example
//
// Do not build by default.

Reply:
fix it commit ee0bff5.